### DISCUSSION TITLE
Preserving the original PDF and making annotations in-place

### DISCUSSION THREAD

**Discussion by lucasrla** [lucasrla/remarks#6] [General]
Following up on the conversation inside issue #2, I have pushed a new commit ([99ec38](https://github.com/lucasrla/remarks/commit/99ecc382a64a7b92b6a721b6d368f4d0c781c3b2)) to implement an additional enhancement as discussed with @folofjc. 

I will wait for their feedback here.

**Comment by folofjc** [comment]
Hey @lucasrla. I did see this but I have been swamped and haven't had a chance to test it. I will, don't worry, and I will report back. I appreciate you integrating my request!

**Comment by folofjc** [comment]
Hey @lucasrla I (finally, sorry!) got a chance to look at it. Yep, the links still work on pages that I annotated! That is great, that is a TON of help. I just need to remember to not write near the edges of the document so that it won't have to be resized. I know that some people just add an annotation layer and write on the page and then hide it when they want to read. I might have to start doing that, haha. Actually, better yet, if I want to keep the links, I can export only the highlights, since that won't resize the page. Then I can have a separate one where I pull out the scribbles? Maybe.

Thanks a ton for the help and for adding this functionality!

**Reply by lucasrla** [reply]
Good to hear that it is working well. You are welcome!

Regarding exporting just the highlights to PDF, it is something that, in principle, seems very doable. Do you mind opening an issue with a feature request?

Thanks
